packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5964,6 +5969,20 @@ export default class Meeting extends StatelessWebexPlugin {
         return leave;
       })
       .catch((error) => {
+        // CA team recommends submitting this *after* locus /leave
+        submitLeaveMetric({
+          errors: [
+            {
+              fatal: false,
+              errorDescription: error.message,
+              category: 'signaling',
+              errorCode: 1000,
```

Comment:
do we know what 1000 means? if yes, could you create a const in https://github.com/torgeadelin/webex-js-sdk/blob/6374853f575595392d02a777db3ef96642b86c14/packages/@webex/internal-plugin-metrics/src/call-diagnostic/config.ts#L8